### PR TITLE
Remove mobile

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* updated the homepage text and images
* updated the projects page
* updated the partners' text and images
* updated the projects page
* updated the careers references

## QA

*see that the images have to mobile devices - ALL PAGES BELOW*

1. go to / and see that the copy matches the [copy doc](https://docs.google.com/document/d/1WcNN2RlQQrbLSP5ZgcIsAeQAHj5h_yDW6Yd5oBMmU3M/edit#)
2. go to /products and see the copy matches the [copy doc](https://docs.google.com/document/d/1-PY9w_BtCGS9CkSJqTKjOCkSzmgvROxfk-BcwitFEcE/edit)
3. go to /partners and see the copy matches the [copy doc](https://docs.google.com/document/d/1O7rMRIcRVINNsoiJ5fUQ4FPqUxGDKB3E6AaPZU2BAmc/edit#)
4. go to the vacancies pages and see they do not mention mobile

## Demo

[demo website](http://www.canonical.com-remove-mobile.demo.haus/)

### CODE REVIEW

**Comment by yaili** [comment]
Some notes:
- updated image on homepage should be vertically centred in the row (medium, large screens)
  - on small screens the image is too close to the bottom border of the row
- the updated products laptop image has a white non-transparent bottom - do we need to find a new one?
- partners and careers look ok

**Comment by anthonydillon** [comment]
## QA issues
The products page
- missing the "On PCs : supported Ubuntu desktop" section from the copy doc
- It has an extra section of text below the laptop image in the "On the client: devops for IoT" section. Which is not in the copy doc

File: templates/products/index.html [hunk 1]
```diff
@@ -100,16 +100,18 @@
...
 	<div class="inner-wrapper">
 	    <div class="twelve-col">
-	        <h2>On the client&#58; multiple devices, one experience</h2>
+	        <h2>On the client&#58; devops for IoT</h2>
 	    </div>
 	    <div class="eight-col">
-	        <p>Ubuntu is a single software platform that runs across smartphones, tablets, PCs, and TVs. It is designed to help make converged computing a reality&#58; one system, one experience, multiple form factors.</p>
+        <p>Ubuntu is the preferred development environment for embedded engineers and widely used in <a href="https://www.ubuntu.com/internet-of-things">IoT devices</a> across verticals: automotive, industrial, robotics, networking and digital signage. Canonical works with semiconductors to make certified Ubuntu images that brings the best performance out of their hardware.</p>
+        <p>With snaps, the universal Linux packaging system, and <a href="https://snapcraft.io">snapcraft.io</a> developers a tool to build and distribute their software across multiple devices to facilitate the deployment of applications in production environments.</p>
+        <p>Ubuntu Core is the all snap version of Ubuntu, built for security.</p>
```
Comment (anthonydillon) [inline templates/products/index.html]: Under tabbed.

Comment (pmahnke) [inline templates/products/index.html]: fixed

**Comment by pmahnke** [comment]
@anthonydillon thanks... I added that **On PCs** section ... I don't know how I missed that.

**Pull request merged**